frontend/packages/shared/src/types/ResourceAdm.ts

```diff
@@ -1,18 +1,20 @@
+import { SupportedLanguage, ValidLanguage } from 'resourceadm/types/global';
```

Comment:
We should not do any imports from other packages into shared. Shared is supposed to be a folder where all the other packages has their shared code, and to prevent circular dependencies we aim to avoid other imports 😄 
If you need these two types in this file, I would suggest moving them to this file instead 😄